$OpenBSD: patch-Source_CPack_cmCPackGenerator_cxx,v 1.3 2014/09/28 18:15:41 dcoppa Exp $

Use GetSafeDefinition so that a missing definition is treated as
an empty string (adapted from upstream git commit
3bae343f62e050da78476811234e1f8a055f5d04)

--- Source/CPack/cmCPackGenerator.cxx.orig	Sun Sep 28 19:26:21 2014
+++ Source/CPack/cmCPackGenerator.cxx	Sun Sep 28 19:29:47 2014
@@ -637,7 +637,7 @@ int cmCPackGenerator::InstallProjectViaInstallCMakePro
         {
         globalGenerator->FindMakeProgram(this->MakefileMap);
         const char* cmakeMakeProgram
-          = this->MakefileMap->GetDefinition("CMAKE_MAKE_PROGRAM");
+          = this->MakefileMap->GetSafeDefinition("CMAKE_MAKE_PROGRAM");
         std::vector<std::string> buildCommand;
         globalGenerator->GenerateBuildCommand(buildCommand, cmakeMakeProgram,
             installProjectName.c_str(), installDirectory.c_str(),
